polaris/dataset/_column.py

```diff
@@ -33,10 +39,11 @@ class ColumnAnnotation(BaseModel):
     """
 
     is_pointer: bool = False
-    modality: Union[str, Modality] = Modality.UNKNOWN
-    description: Optional[str] = None
+    modality: str | Modality = Modality.UNKNOWN
+    description: str | None = None
     user_attributes: Dict[str, str] = Field(default_factory=dict)
-    dtype: Optional[Union[np.dtype, str]] = None
+    dtype: np.dtype | str | None = None
+    content_type: str | ContentType | None = None
```

Comment:
**Suggestion:** Include this newly added field in the docstring for the class.

I can imagine that having to set the content type, especially to something specific like `chemical/x-daylight-smiles`, is something the average ML engineer won't be used to or expect. Documenting the field is a first step, but it makes me wonder if there's other ways through which we could make for a better UX. E.g. allowing people to edit this field on the Hub!